eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/configuration/SwaggerConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ebi.eva.accession.ws.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import static springfox.documentation.builders.PathSelectors.regex;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfiguration {
+
+    @Bean
+    public Docket productApi() {
+        return new Docket(DocumentationType.SWAGGER_2)
+                .select()
+                .apis(RequestHandlerSelectors.basePackage("uk.ac.ebi.eva.accession.ws.rest"))
+                .paths(regex("/v1/submitted-variants.*"))
```

Comment:
Why not .paths(PathSelectors.any()) ? Are there any paths that we want to exclude?